Dropdown/Dropdown.js

```diff
@@ -124,6 +124,8 @@ const DropdownBase = kind({
 	name: 'Dropdown',
 
 	propTypes: /** @lends agate/Dropdown.DropdownBase.prototype */ {
+		adjusteddirection: PropTypes.oneOf(['above', 'below']),
```

Comment:
Please add default value of this prop

Reply:
I accepted your suggestion.
The default value is 'below'.